.golangci.yml

```diff
@@ -0,0 +1,38 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+run:
+  concurrency: 4
+  timeout: 5m
+  issues-exit-code: 0 # TODO: once all enabled linters are satisfied  
+  tests: false
+  skip-dirs:
+    - ^experimental.*($|/)
+  skip-dirs-use-default: true
+
+  output:
+  # colored-line-number|line-number|json|tab|checkstyle|code-climate, default is "colored-line-number"
+    format: colored-line-number
+    print-issued-lines: true
+    print-linter-name: true
+
+linters:
+  disable-all: true
+  enable:
+    - govet
+    - errcheck
+    - structcheck
+    - golint
+    - gofmt
+    - staticcheck
```

Comment:
Is there a way to configure these linters? What do they all check?

Reply:
we can configure them in this file under the key `linters-settings` but I thought leaving defaults would be a first good start.

What they check can be seen at https://github.com/golangci/golangci-lint#supported-linters or list a small description and link to the linter itself